### ISSUE TITLE
Blank screen when running app on microsoft remote desktop

### ISSUE THREAD

**Issue by clbsoft** [opentk/GLWpfControl#42]
I tested out the wpf control and it works great locally. I tried remoting in to my work computer with remote desktop and when I run it the window is blank white. It doesn't crash but you can't see anything. Both computers have Windows 10 and nvidia graphics cards with the latest drivers. Also remote desktop worked fine on the old winforms gl control.

**Comment by marcotod1410** [comment]
By default, the rendering over RDP is not hardware accelerated. For this reason, OpenGL does not work over RDP. However, you could enable GPU rendering over RDP enabled on the remote machine to overcome this issue.

Try enable it by following [this procedure](https://community.esri.com/t5/implementing-arcgis-questions/enabling-gpu-rendering-on-windows-server-2016-windows-10-rdp/td-p/658522) and let us know if it works!

**Comment by clbsoft** [comment]
In most cases I wouldn't be able to change group policy for our rdp computers. However I believe my remote desktop is running in hardware mode because calling GL.GetString(StringName.Renderer) returns Geforce GTX 1060 which is the graphics card on the remote machine. Also I tried copying an openGL software driver to the executing directory but this causes the program to crash on load with System.AccessViolationException

![image](https://user-images.githubusercontent.com/79230131/108404010-5ad49080-71ed-11eb-8e61-a0b1e8733ebc.png)

**Comment by marcotod1410** [comment]
Good, it seems that OpenGL is loaded correctly.
I guess that the AccessViolationException you're encountering when using the software renderer is correct: the control is based on a hidden GLFW window that doesn't support software rendering, as far as I know.

**Comment by clbsoft** [comment]
I did some more digging and found a solution. Apparently the issue is on the Direct3D side of things. In the source file GLWpfControlRenderer.cs there is a method PostRender(). If you change the line:
_framebuffer.D3dImage.SetBackBuffer(D3DResourceType.IDirect3DSurface9, _framebuffer.DxRenderTargetHandle);
To this:
_framebuffer.D3dImage.SetBackBuffer(D3DResourceType.IDirect3DSurface9, _framebuffer.DxRenderTargetHandle, true);
RDP will work. The name of the argument I set to true is enableSoftwareFallback. Not sure if it will impact performance in non RDP scenarios. Maybe you will want to make it a property on the control that users can toggle if they are running RDP.
